app/serializers/comment_serializer.rb

```diff
@@ -1,12 +1,18 @@
 class CommentSerializer < ActiveModel::Serializer
   include TagHelper
 
-  attributes :id, :content, :markdown, :commentable_type, :commentable_id, :owner_id, :created_at, :updated_at, :team_id, :owner
+  attributes :id, :content, :markdown, :commentable_type, :commentable_id, :owner_id, :created_at, :updated_at, :team_id, :owner, :attachments
 
   def owner
     ActiveModel::SerializableResource.new(object.owner).serializable_hash[:user]
   end
 
+  def attachments
```

Comment:
Has happened me in a few places as well, tried to read the source code of AMS to figure out what situations it loads in the `has_many`s or when it doesn’t, I’ve still no clue. Kinda why I’d consider using that [jbuilder](https://github.com/rails/jbuilder) thing to build up JSON, even though it’s more verbose it’s much easier to understand what’s actually going on and you don’t have any headaches to figure out when the magic acts up.